SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -207,6 +211,10 @@ void USpatialNetDriver::InitiateConnectionToSpatialOS(const FURL& URL)
 		// Create a new SpatialWorkerConnection in the SpatialGameInstance.
 		GameInstance->CreateNewSpatialWorkerConnection();
 	}
+	else
+	{
+		UE_LOG(LogSpatialOSNetDriver, Log, TEXT("Getting existing connection, not creating a new one"));
```

Comment:
Hmmm, I dunno if there should be a log here. If there is I think I would prefer the log to be the suggestion I put.

```suggestion
		UE_LOG(LogSpatialOSNetDriver, Display, TEXT("Reusing existing SpatialWorkerConnection"));
```

Reply:
✓